packages/application-shell/src/components/navbar/menu-items.tsx

```diff
@@ -246,7 +246,7 @@ type MenuItemProps = {
 };
 const MenuItem = (props: MenuItemProps) => (
   <li
-    role="menu-item"
+    role="menuitem"
```

Comment:
RTL `findByRole` wouldn't work otherwise. 
([MDN ref](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role))
<img width="795" alt="Screenshot 2023-04-24 at 15 39 17" src="https://user-images.githubusercontent.com/49066275/234015419-10834afa-e81f-4a8c-a9ba-4fff36f76097.png">